components/insight/SRC/org/openmicroscopy/shoola/util/ui/slider/TwoKnobsSliderModel.java

```diff
@@ -280,7 +280,7 @@ void setStartValue(double startValue)
 
 	/**
 	 * Sets the value of the end knob, value in the range
-	 * <code>]{@link #startValue}, {@link #maximum}]</code>.
+	 * <code>]{@link #startValue}, {@link #maximum}]  (one based) </code>.
```

Comment:
It turns out you're right, it's a French notation I'd never seen before: http://hsm.stackexchange.com/questions/142/why-is-american-and-french-notation-different-for-open-intervals-x-y-vs-x so it at least has precedent. :+1: